quick-start.md

```diff
@@ -2,222 +2,87 @@
 
-If you followed the steps above, you should now have a working Kontena setup installed. Verify the setup using the `kontena node list` command. It should list all the Kontena Nodes in your Grid.
+If you followed the steps above, you should now have a working Kontena setup installed. Verify the setup using the `node list` command. It should list all the Kontena Nodes in your Kontena Platform.
 
 ```
-$ kontena node list
-NAME              VERSION   STATUS   INITIAL   LABELS
-⊛ damp-forest-2   1.3.4     online   1 / 1     provider=vagrant
+> node list
+NAME              VERSION   STATUS       INITIAL   LABELS
+⊛ damp-forest-2   1.4.0     online 11s   1 / 1     region=eu-west-1,az=a,provider=kontena
 ```
 
 ## Step 4. Deploy Your First Application Stack
 
- Now you are ready to deploy your first application stack.
- In this section we will show you how to package a simple WordPress application and deploy it to your Kontena Grid.
-
-First create the `kontena.yml` file with the following contents:
-
-```yaml
-stack: examples/wordpress
-version: 0.3.0
-variables:
-  wordpress-mysql-root:
-    type: string
-    from:
-      vault: wordpress-mysql-root
-      random_string: 32
-    to:
-      vault: wordpress-mysql-root
-  wordpress-mysql-password:
-    type: string
-    from:
-      vault: wordpress-mysql-password
-      random_string: 32
-    to:
-      vault: wordpress-mysql-password
-services:
-  wordpress:
-    image: wordpress:4.6
-    stateful: true
-    ports:
-      - 80:80
-    environment:
-      WORDPRESS_DB_HOST: mysql
-      WORDPRESS_DB_USER: wordpress
-      WORDPRESS_DB_NAME: wordpress
-    secrets:
-      - secret: wordpress-mysql-password
-        name: WORDPRESS_DB_PASSWORD
-        type: env
-  mysql:
-    image: mariadb:5.5
-    stateful: true
-    environment:
-      MYSQL_DATABASE: wordpress
-      MYSQL_USER: wordpress
-    secrets:
-      - secret: wordpress-mysql-root
-        name: MYSQL_ROOT_PASSWORD
-        type: env
-      - secret: wordpress-mysql-password
-        name: MYSQL_PASSWORD
-        type: env
-```
+After the provisioned node is online, you are ready to deploy your first application stack. To install a ready-made stack you can execute `stack install kontena/hello-world` command in terminal.
 
-You can then install and deploy the `wordpress` stack:
-
-```
-$ kontena stack install kontena.yml
- [done] Creating stack wordpress
- [done] Deploying stack wordpress
-```
+<img src="_images/kontena-cloud-install-stack.png" alt="Install Stack" />
 
 The initial stack deployment may take some time while the host nodes pull the referenced Docker images.
 
-After the stack deployment is finished you can verify that the wordpress and mysql services are running:
+After the stack deployment is finished you can verify from Kontena Cloud Dashboard that the lb and web services are running or executing `stack ls` command:
```

Comment:
After the stack deployment is finished you can verify from the Kontena Cloud Dashboard that the lb and web services are running or executing the `stack ls` command: